react-context/client/src/pages/HomePage.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-unused-vars -- Remove me */
+/* eslint-disable no-undef -- Remove me */
+import { useContext, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import AppContext from '../components/AppContext';
+import './HomePage.css';
+
+export default function Home() {
+  /* TODO: Grab `user` from `AppContext` */
+  const user = useContext(AppContext);
```

Comment:
This isn't correct. You are getting the entire AppContext value, not just the user. You will always get a value back, which means you don't correctly detect if the user is logged in.